examples/ink/interact.eg.ts

```diff
@@ -2,13 +2,12 @@
  * @title Interact With An Ink Smart Contract
  * @stability unstable – We intend to work on an Ink provider (for static codegen)
  * in the near future. This work will likely entail large changes to the current ink patterns.
- *
- * The Ink patterns simplify the reading of contract instance state and events, as well as
+ * @description The Ink patterns simplify the reading of contract instance state and events, as well as
  * the submission of transactions.
  */
 
-import { assertNotEquals } from "asserts"
-import { alice } from "capi"
+import { assert } from "asserts"
+import { $, alice, bob } from "capi"
```

Comment:
Should use test users

Reply:
This is one of those unique cases where a constant test user is preferable: the contract is deployed from Alice, hence only Alice has funds to transfer. We could do this with `createUser`, but it would involve merging the deploy and interact examples. I'm not opposed to this, but let's perhaps do this at a later date.

Reply:
In that case, can you put that context in a comment?